packages/toolkit/src/query/fetchBaseQuery.ts

```diff
@@ -27,6 +27,13 @@ export interface FetchArgs extends CustomRequestInit {
   validateStatus?: (response: Response, body: any) => boolean
 }
 
+/**
+ * A mini-wrapper that passes arguments straight through to `window.fetch`.
+ * Avoids storing `window.fetch` in a closure, in order to permit mocking/monkey-patching.
+ */
+const defaultFetchFn = (...args: Parameters<typeof window.fetch>) =>
+  window.fetch(...args)
```

Comment:
```suggestion
const defaultFetchFn: typeof fetch = (...args) =>
  fetch(...args)
```

Given that we had it without the `window` reference before, let's just keep it like that.

Reply:
No problem, changed as per the above, and tweaked the docstring slightly.

The reason I explicitly changed it previously was that I've experienced a setup where Sinon's [useFakeTimers](https://sinonjs.org/releases/latest/fake-timers/) will mock `window.setTimeout` but not `setTimeout`. But given that I haven't found the exact reason for that, or how version specific or test setup specific the behaviour is, I can't assume that will occur elsewhere.